### DISCUSSION TITLE
Unable to open ios RCTMGL.xcodeproj using XCODE 12.4 (12D4e)

### DISCUSSION THREAD

**Discussion by muralee** [rnmapbox/maps#1507] [Q&A]
Hi,

I forked master and trying to add new feature to transform tile request urls (my case i need to transform the url with TTL based token) but i could not open xcodeproj file to add new features. I am getting following error

The project ‘RCTMGL’ is damaged and cannot be opened due to a parse error. Examine the project file for invalid edits or unresolved source control conflicts.

Path: /Users/XXXX/Downloads/maps-master/ios/RCTMGL.xcodeproj

Can you please help me to fix this error?

Thank you

**Comment by mfazekas** [comment]
@muralee not sure if you(or we) need that RCTMGL.xcodeproj I think it can be removed, cocoa pods should work.

**Reply by muralee** [reply]
@mfazekas I tried pod install from terminal but no luck (since no Podfile in ios folder) and really i have no idea how to open it (i can open iOS in example directory) but not the one is main

It will be great if you can help me to solve may be i am missing something

Thanks

**Reply by muralee** [reply]
@mfazekas by opening example ios project i can add/edit files in Pods > Development Pods > react-native-mapbox-gl > DynamicLibrary and it works for me. But i don't know is it correct way or not

just wanted to inform you that now i can able to extend the feature 

thanks
